mrmustard/math/math_interface.py

```diff
@@ -377,14 +377,18 @@ def hash_tensor(self, tensor: Tensor) -> int:
         """
 
     @abstractmethod
-    def hermite_renormalized(self, A: Matrix, B: Vector, C: Scalar, shape: Sequence[int]) -> Tensor:
+    def hermite_renormalized(
+        self, A: Matrix, B: Vector, C: Scalar, shape: Sequence[int], modified: bool
+    ) -> Tensor:
```

Comment:
why do we need the unmodified Hermite polynomials?

Reply:
removed

Reply:
For homodyne sampling I use the unmodified Hermite polynomials. I'll make it more explicit in the code.